dive_core/gfxr_vulkan_command_hierarchy.cpp

```diff
@@ -46,6 +46,24 @@ DiveAnnotationProcessor::VulkanCommandInfo vk_cmd_info)
                  m_cur_submit_node_index,
                  cmd_buffer_index);
     }
+    else if (vk_cmd_info.GetVkCmdName().find("vkCmdDraw") != std::string::npos)
```

Comment:
do you need other draw & dispatch cmds?
vkCmdDrawIndirect
vkCmdDrawIndexed
vkCmdDrawIndexedIndirect
vkCmdDispatch
vkCmdDispatchIndirect

Reply:
Thanks